system-int-test/src/cmr/system_int_test/data2/atom_json.clj

```diff
@@ -62,6 +62,11 @@
   (when (and v (seq v))
     (Double. v)))
 
+(defn parse-integer
+  [^String v]
+  (when (and v (seq v))
```

Comment:
`(when-not (string/blank? v)` seems more intuitive, but looks like you are just copying existing functions.

Reply:
Changed for all parsing functions.